shell/key-bindings.bash

```diff
@@ -132,9 +133,14 @@ if ((BASH_VERSINFO[0] < 4)); then
   fi
 
   # CTRL-R - Paste the selected command from history into the command line
-  bind -m emacs-standard '"\C-r": "\C-e \C-u\C-y\ey\C-u`__fzf_history__`\e\C-e\er"'
-  bind -m vi-command '"\C-r": "\C-z\C-r\C-z"'
-  bind -m vi-insert '"\C-r": "\C-z\C-r\C-z"'
+  if [[ ${FZF_CTRL_R_COMMAND-x} != "" ]]; then
+    if [[ -n ${FZF_CTRL_R_COMMAND+x} && -n $FZF_CTRL_R_COMMAND ]]; then
```

Comment:
Is the former condition required? Shouldn't `-n ${FZF_CTRL_R_COMMAND-}` suffice?

Reply:
I will, and it's also easier to reason about. Will change.

Reply:
I also learned that Fish (not a shell I use) is more permissive with unset variables, and so I simplified the condition there.

Reply:
Thanks. Reference: https://github.com/fish-shell/fish-shell/issues/4163